CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# v0.0.34...v0.0.35
```

Comment:
Looks good 😁 but do you want a top level H1, e.g. `# Change Log`?

Reply:
good call, just pushed that